### PR TITLE
Fix Interface conversion error on oc.go

### PR DESCRIPTION
**Pull request by mmulholla**

### CODE REVIEW

**Comment by baijum** [comment]
To avoid further regression, let's add some unit tests.

**Comment by mmulholla** [comment]
Test looks good. Thank you Baiju.

File: pkg/tool/oc.go [hunk 1]
```diff
@@ -41,8 +41,8 @@ func (o Oc) GetVersion() (string, error) {
 	}
 	// Relying on Kubernetes version can be replaced after fixing this issue:
 	// https://bugzilla.redhat.com/show_bug.cgi?id=1850656
-	kubeServerVersion := out[osVersionKey].(map[string]string)
-	kubeVersion := kubeServerVersion["major"] + "." + kubeServerVersion["minor"]
+	kubeServerVersion := out[osVersionKey].(map[string]interface{})
+	kubeVersion := fmt.Sprintf("%v.%v", kubeServerVersion["major"], kubeServerVersion["minor"])
```
Comment (baijum) [inline pkg/tool/oc.go:45]: Instead of `%v` format string, I think `%s` would be more appropriate.

**Review by baijum — APPROVED** [review]
LGTM

File: pkg/tool/oc.go (hunk 1 above)
Comment (mmulholla) [reply pkg/tool/oc.go]: Updated

**Pull request merged**